core/src/trezor/sdcard.py

```diff
@@ -26,13 +31,24 @@ def _deinit_instance(self) -> None:
         sdcard.power_off()
         FilesystemWrapper._INSTANCE = None
 
+    def _try_mount(self) -> None:
+        """Try to mount filesystem, raise appropriate exception."""
```

Comment:
I'd appreciate a better comment here. Something like:
_Try to mount filesystem **if a mounted instance was requested**. Raise appropriate exception._
Otherwise the if-statement on the next line is confusing to someone who is not familiar with the logic (the `not` in the condition looks like a bug).

Reply:
this code is now removed